src/duration.rs

```diff
@@ -359,6 +391,27 @@ impl fmt::Display for Duration {
     }
 }
 
+/// Represents error when converting `Duration` to/from a standard library
+/// implementation
+///
+/// The `std::time::Duration` supports a range from zero to `u64::MAX`
+/// *seconds*, while this module supports signed range of up to
+/// `i64::MAX` of *milliseconds*.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct OutOfRangeError;
```

Comment:
Could you make this `struct OutOfRangeError(())` so we can extend it in the future as well?